tox.ini

```diff
@@ -30,8 +30,11 @@ deps =
 commands =
     {envpython} --version
     trial --version
-    coverage run --source klein --omit klein/test/* --branch {envdir}/bin/trial {posargs:klein}
+    coverage erase
```

Comment:
Here's how Mimic does it: https://github.com/rackerlabs/mimic/blob/bd6620b3dd6576986177abb090317e27d5dea205/.travis.yml#L52-L53